COLLABORATOR_GUIDE.md

```diff
@@ -88,6 +88,11 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+If any Collaborator objects to a change without giving any additional
+explanation or context, and the objecting Collaborator fails to respond to
+explicit requests for explanation or context within a reasonable period of
+time, the objection may be dismissed.
```

Comment:
Not a blocking objection on this, but a suggested enhancement: Specify a length of time:

> and the objecting Collaborator fails to respond to
> explicit requests for explanation or context for more than one week,
> the objection may be dismissed.